src/assets/scss/_settings.scss

```diff
@@ -207,14 +207,20 @@ $header-styles: (
 	small: (
 		'h1': ('font-size': rem-calc(28), 'line-height': rem-calc(36) / rem-calc(28)),
 		'h2': ('font-size': rem-calc(22), 'line-height': rem-calc(30) / rem-calc(22)),
```

Comment:
This tells foundation to generate .h3, and .h4 classes in the same way it was generating .h1 and .h2 classes.